parachain/src/service.rs

```diff
@@ -322,7 +325,7 @@ where
 
         cumulus_client_consensus_aura::import_queue::<AuraPair, _, _, _, _, _>(
             cumulus_client_consensus_aura::ImportQueueParams {
-                block_import: ParachainBlockImport::new(client.clone(), backend.clone()),
+                block_import: parachain_block_import.clone(),
```

Comment:
Not sure if this clone is problematic.. It seems to have worked for https://github.com/moonbeam-foundation/moonbeam/pull/2470/files#diff-6dcbafad4c91afb2c91f5b01429549242a45a3bb0de721c830abd0a61421ae3aR529 like this

Reply:
Should be fine